test/browser_tests/conf.js

```diff
@@ -151,77 +151,88 @@ function _copyParameters( params, capabilities ) { // eslint-disable-line comple
+    delete config.sauceUser;
+    delete config.sauceKey;
+  }
 
-    var suite = _chooseSuite( params );
-    var capabilities = _copyParameters( params, suite );
-    return capabilities;
-  },
-
-  onPrepare: function() {
-    // Ignore Selenium allowances for non-angular sites.
-    browser.ignoreSynchronization = true;
-
-    // If --windowSize=w,h flag was passed, set window dimensions.
-    // Otherwise, use default values from the test settings.
-    var windowSize = browser.params.windowSize;
-    var windowWidthPx;
-    var windowHeightPx;
-    if ( typeof windowSize === 'undefined' ) {
-      windowWidthPx = environment.windowWidthPx;
-      windowHeightPx = environment.windowHeightPx;
-    } else {
-      var windowSizeArray = windowSize.split( ',' );
-      windowWidthPx = Number( windowSizeArray[0] );
-      windowHeightPx = Number( windowSizeArray[1] );
-    }
+  const suite = _chooseSuite( params );
+  const capabilities = _copyParameters( params, suite );
+  return capabilities;
+}
 
-    // Calling setSize with headless chromeDriver doesn't work properly if
-    // the requested size is larger than the available screen size.
-    if ( !envvars.HEADLESS_CHROME_BINARY ) {
-      browser.driver.manage().window().setSize(
-        windowWidthPx,
-        windowHeightPx
-      );
-
-      // Set default windowSize parameter equal to the value in settings.js.
-      browser.params.windowWidth = windowWidthPx;
-      browser.params.windowHeight = windowHeightPx;
-      browser.params.windowSize = String( windowWidthPx ) +
-                                  ',' + String( windowHeightPx );
-    }
+/**
+ * The onPrepare method for Protractor's workflow.
+ * See https://github.com/angular/protractor/blob/master/docs/system-setup.md
+ */
+function _onPrepare() {
+  // Ignore Selenium allowances for non-angular sites.
+  browser.ignoreSynchronization = true;
+
+  // If --windowSize=w,h flag was passed, set window dimensions.
+  // Otherwise, use default values from the test settings.
+  const windowSize = browser.params.windowSize;
+  let windowWidthPx;
+  let windowHeightPx;
+  if ( typeof windowSize === 'undefined' ) {
+    windowWidthPx = environmentTest.windowWidthPx;
+    windowHeightPx = environmentTest.windowHeightPx;
+  } else {
+    const windowSizeArray = windowSize.split( ',' );
+    windowWidthPx = Number( windowSizeArray[0] );
+    windowHeightPx = Number( windowSizeArray[1] );
+  }
 
-    return;
+  // Calling setSize with headless chromeDriver doesn't work properly if
+  // the requested size is larger than the available screen size.
+  if ( !envvars.HEADLESS_CHROME_BINARY ) {
+    browser.driver.manage().window().setSize(
+      windowWidthPx,
+      windowHeightPx
+    );
+
+    // Set default windowSize parameter equal to the value in settings.js.
+    browser.params.windowWidth = windowWidthPx;
+    browser.params.windowHeight = windowHeightPx;
+    browser.params.windowSize = String( windowWidthPx ) +
```

Comment:
I might leave it since it makes it clearer that the instances of `windowWidthPx` and `windowHeightPx` on the lines above it are numbers not strings. I didn't update this, that's verbatim what it was before I moved it.